src/WebJobs.Extensions.DurableTask/Correlation/TelemetryActivator.cs

```diff
@@ -15,12 +17,14 @@ namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Correlation
     /// <summary>
     /// TelemetryActivator initializes Distributed Tracing. This class only works for netstandard2.0.
     /// </summary>
-    public class TelemetryActivator : ITelemetryActivator
+    public class TelemetryActivator : ITelemetryActivator, IAsyncDisposable
     {
         private readonly DurableTaskOptions options;
-        private TelemetryClient telemetryClient;
+        private readonly INameResolver nameResolver;
         private EndToEndTraceHelper endToEndTraceHelper;
-        private INameResolver nameResolver;
+        private TelemetryClient telemetryClient;
+        private IAsyncDisposable disposable;
```

Comment:
Any reason to not just call this what it is?
```suggestion
        private IAsyncDisposable telemetryModule;
```